pom.xml

```diff
@@ -1,23 +1,771 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.openhab.addons.tmp</groupId>
-  <artifactId>org.openhab.addons.tmp</artifactId>
+  <groupId>org.openhab.addons</groupId>
+  <artifactId>org.openhab.addons.reactor</artifactId>
   <version>2.5.0-SNAPSHOT</version>
+
   <packaging>pom</packaging>
-  <name>openHAB 2 Add-ons</name>
```

Comment:
Why did you change from "OpenHAB 2 Add-ons" to "OpenHAB Add-Ons"?

Reply:
I moved the poms/bnd/pom.xml file to root and it was defined like this by Markus I guess.
If the guys over at OH1-addons don't start migrating they will not have a working IDE setup anymore in a few days, so we don't actually have OH1 vs OH2 anymore and could also just say "OpenHAB Add-Ons" here.